src/pages/feedback.vue

```diff
@@ -97,16 +97,12 @@ export const FeedbackPage = {
     },
   },
   computed: {
-    ...mapState(['isEmbedded']),
-    isReportingBug() {
-      return this.$store.state.isReportingBug
-    },
-    bugReported() {
-      return this.$store.state.bugReported
-    },
-    bugReportFailed() {
-      return this.$store.state.bugReportFailed
-    },
```

Comment:
Just a curiosity, but why were these not using `mapState` before?

Reply:
I am not sure, probably because that was the understanding at the time of writing of this code.